custom/icds_reports/migrations/0108_child_cases_ccs_record_supervisor_id.py

```diff
@@ -9,7 +9,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('icds_reports', '0106_left_join_service_delivery'),
+        ('icds_reports', '0107_citus_composite_key'),
```

Comment:
@snopoke fyi.
Existing Master code would fail on deployment without this merged